integration-tests/smoke/vrfv3_test.go

```diff
@@ -0,0 +1,336 @@
+package smoke
+
+import (
+	"fmt"
+	"github.com/smartcontractkit/chainlink-env/environment"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/chainlink"
+	eth "github.com/smartcontractkit/chainlink-env/pkg/helm/ethereum"
+	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
+	"github.com/smartcontractkit/chainlink/core/services/keystore/chaintype"
+	"github.com/smartcontractkit/chainlink/core/utils"
+	networks "github.com/smartcontractkit/chainlink/integration-tests"
+	"github.com/smartcontractkit/chainlink/integration-tests/actions"
+	"github.com/smartcontractkit/chainlink/integration-tests/client"
+	"github.com/smartcontractkit/chainlink/integration-tests/contracts"
+	ocr2vrftypes "github.com/smartcontractkit/ocr2vrf/types"
+	"github.com/stretchr/testify/require"
+	"math/big"
+	"strings"
+	"testing"
+
+	"github.com/rs/zerolog/log"
+)
+
+func TestVRFv3Basic(t *testing.T) {
```

Comment:
```suggestion
func TestOCR2VRFBasic(t *testing.T) {
```

Reply:
So you are suggesting omitting the V3 notation? I thought to simplify things here and it's a bit more concise

Reply:
Yes, OCR2VRF is the operative name and that's what we use in the rest of the codebase.

Reply:
renamed